synapse/federation/transport/client.py

```diff
@@ -104,7 +108,7 @@ async def get_event(
 
     @log_function
     async def backfill(
-        self, destination: str, room_id: str, event_tuples: Iterable[str], limit: int
+        self, destination: str, room_id: str, event_tuples: List[str], limit: int
```

Comment:
It'd be nice to have a type definition for "things that can be json-serialised into a list", which would include Tuples as well as Lists.

not a blocker though.